### PR TITLE
update roadmap

### PR DESCRIPTION
**Pull request by kadel**
info about monthly releases

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -5,15 +5,9 @@ This document is a general overview of all upcoming development for `kompose`. T
 We track each release by using GitHub [milestones](https://github.com/kubernetes-incubator/kompose/milestones).
 
 ## Upcoming releases
+Kompose is released every the last week of a month.
```
Comment (cdrage) [inline ROADMAP.md]: `of the month`

File: ROADMAP.md [hunk 2]
```diff
@@ -5,15 +5,9 @@ This document is a general overview of all upcoming development for `kompose`. T
...
 
-#### kompose 0.3.0
-
-[https://github.com/kubernetes-incubator/kompose/milestone/1](https://github.com/kubernetes-incubator/kompose/milestone/1)
-
-#### kompose 0.2.1
-
-[https://github.com/kubernetes-incubator/kompose/milestone/2](https://github.com/kubernetes-incubator/kompose/milestone/2)
-
+You can see all planing releases on GitHub [milestones](https://github.com/kubernetes-incubator/kompose/milestones) page.
```
Comment (cdrage) [inline ROADMAP.md]: `You can see all planned releases on the GitHub milestones page`

File: ROADMAP.md (hunk 1 above)
Comment (kadel) [inline ROADMAP.md]: oh, I had it there and than I changed it :-D

File: ROADMAP.md (hunk 2 above)
Comment (kadel) [inline ROADMAP.md]: I "love" articles in English  :-D

File: ROADMAP.md (hunk 1 above)
Comment (cdrage) [inline ROADMAP.md]: woops. it's suppose to be `Kompose is released every last week of the month`

File: ROADMAP.md [hunk 3]
```diff
@@ -5,15 +5,9 @@ This document is a general overview of all upcoming development for `kompose`. T
 We track each release by using GitHub [milestones](https://github.com/kubernetes-incubator/kompose/milestones).
 
 ## Upcoming releases
+Kompose is released every the last week of the month.
```
Comment (cdrage) [inline ROADMAP.md]: I think it should be:
`Kompose is released every last week of the month`

Otherwise, LGTM! :) Merge away when done!

**Comment by kadel** [comment]
Thank you @cdrage.

**Pull request merged**